project-base/storefront/components/Pages/CategoryDetail/CategoryBestsellers/CategoryBestsellersListItem.tsx

```diff
@@ -31,46 +31,39 @@ export const CategoryBestsellersListItem: FC<CategoryBestsellersListItemProps> =
             href={productUrl}
             type={product.__typename === 'RegularProduct' ? 'product' : 'productMainVariant'}
             className={twJoin(
-                'flex flex-wrap items-center gap-y-4 p-4 rounded lg:flex-nowrap lg:gap-5 transition-colors no-underline hover:no-underline',
+                'group flex items-center justify-between gap-5 gap-y-4 p-3 no-underline transition-colors hover:no-underline',
                 'bg-backgroundMore',
-                'hover:bg-backgroundMost',
+                'hover:bg-background',
             )}
             onClick={() => onGtmProductClickEventHandler(product, gtmProductListName, listIndex, url)}
         >
-            <div className="flex items-center gap-5 font-bold no-underline hover:no-underline lg:flex-1">
-                <div className="flex w-20 shrink-0 items-center justify-center">
-                    <Image
-                        alt={product.mainImage?.name || product.fullName}
-                        className="max-h-20 w-auto shrink-0"
-                        height={80}
-                        src={product.mainImage?.url}
-                        tid={TIDs.category_bestseller_image}
-                        width={80}
-                    />
-                </div>
-                <span>{product.fullName}</span>
-            </div>
-
-            <div className="basis-4/6 lg:basis-3/12 lg:text-center">
-                <span
-                    className={twJoin(
-                        product.availability.status === TypeAvailabilityStatusEnum.InStock &&
-                            'text-availabilityInStock',
-                        product.availability.status === TypeAvailabilityStatusEnum.OutOfStock &&
-                            'text-availabilityOutOfStock',
-                    )}
-                >
-                    {product.availability.name}
-                </span>
-
-                <ProductAvailableStoresCount
-                    availableStoresCount={product.availableStoresCount}
-                    isMainVariant={product.isMainVariant}
+            <div className="relative flex w-20 shrink-0 items-center justify-center">
+                <ImageOverlay />
```

Comment:
To be honest I am not a bog fan of the `ImageOverlay` solution. It does not look like in the design and in my opinion it will be a overhead for the developer to keep in mind that the overlay needs to be there in just some places. Let's discuss tomorrow at the UI meeting if we can come up with different solution. Because right now with 20% opacity the difference of the colour is too much IMO.

Reply:
I share the same concerns. I did my best to improve it, but it still doesn't look quite right. We should definitely discuss this further in the upcoming UI meeting.